### ISSUE TITLE
Test runs should use a pre-installed Minikube instance

### ISSUE THREAD

**Issue by mccheah** [apache-spark-on-k8s/spark-integration#4]
#3 makes the tests download a new Minikube installation and deletes the image after every test. This won't work on the Riselab jenkins infrastructure which uses a pre-installed Minikube instance. This also makes the setup unreliable for concurrent builds, as multiple test runs could try to delete each other's Minikube VMs.

We need to incorporate the changes proposed in https://github.com/apache-spark-on-k8s/spark/pull/521 here at some point. However, we need to do this in a specific way to keep builds stable everywhere. The sequence should be:

1. Propose https://github.com/apache-spark-on-k8s/spark/pull/521 as a diff to this repository
2. Turn off builds on the pepperdata jenkins nodes.
3. Merge https://github.com/apache-spark-on-k8s/spark/pull/521 here.
4. Turn on the tests onto the Riselab jenkins nodes.

Basically we can never have it such that https://github.com/apache-spark-on-k8s/spark/pull/521 will run on the Pepperdata jenkins nodes - those tests will surely fail.

**Comment by kimoonkim** [comment]
FYI, @ifilonenko and I chatted in slack. He is porting apache-spark-on-k8s/spark#521, I believe.

**Comment by ifilonenko** [comment]
Can this be closed?
